src/views/Team/About/index.view.tsx

```diff
@@ -0,0 +1,48 @@
+import React from "react"
+import Logo from "../../../images/icons/LogowStars.svg"
+import "./index.scss"
+
+const About: React.FC = () => {
+  return (
+    <div className='about__container'>
+      <div className='about__container--top'>
+        <div className='about__container--logo'>
+          <img src={Logo} alt='CruzHacks Logo' />
```

Comment:
maybe just do `<img className='about__conatiner--logo`... instead of surrounding with divs